api/ws/transport.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var wsApi = require('../../helpers/wsApi');
+var wsRPC = require('./rpc/wsRPC').wsRPC;
+
+function TransportWSApi (transportModule, app, logger) {
+
+	wsRPC.getServer().registerRPCEndpoints({
+		acceptPeer: transportModule.internal.acceptPeer,
+		removePeer: transportModule.internal.removePeer,
```

Comment:
In #654 branch I added divided acceptPeer action into insert and update, so it will be three then. Insert and removed accessible only through peer itself after receiving WS handshake, update after each new block application called by other peers. So they will b changed in corresponding PR created from #654 branch.